labs/a7.md

```diff
@@ -166,13 +172,16 @@ more information.
 
 [lab-a5]: /labs/a5#dns-configuration
```

Comment:
I don't actually see a description of DNS in a5 any more, so probably should remove this